frontend/src/pages/associations/elections/Commons.tsx

```diff
@@ -0,0 +1,40 @@
+import {PageTitle} from "../../../utils/common";
+import React, {useContext, useState} from "react";
+import {Link} from "react-router-dom";
+import {Button, Modal} from "react-bootstrap";
+import Card from "react-bootstrap/Card";
+import Row from "react-bootstrap/Row";
+import {activeStatus, api, useBetterQuery} from "../../../services/apiService";
+import {LoadingAssociation} from "../Loading";
+import {Election, Ballot, Choice} from "../../../models/associations/election";
+import {ToastContext} from "../../../utils/Toast";
+
+export const ListOfVotersButtonModal = ({election, ...props}) => {
```

Comment:
If `Commons` only contains one Element, you could also rename the file after it

Reply:
It contains more than one element now ;)